milli/src/update/index_documents/extract/extract_docid_word_positions.rs

```diff
@@ -70,6 +71,12 @@ pub fn extract_docid_word_positions<R: io::Read + io::Seek>(
                         .take_while(|(p, _)| (*p as u32) < max_positions_per_attributes);
 
                     for (index, token) in tokens {
+                        let script = token.script;
+                        let language = token.language.unwrap_or_default();
+                        let entry = script_language_pair
+                            .entry((script, language))
+                            .or_insert_with(RoaringBitmap::new);
+                        entry.push(document_id);
```

Comment:
I suggest skipping the insertion if the Language is undetected, only the list of detected Language per script is useful at the end

Reply:
I agree. Although it looks like something is missing in charabia because Latin languages are not detected. The results are always (Latin, Other). For the other languages and scripts, the detection works fine.

Reply:
Indeed! Charabia has only one pipeline to handle all Latin Languages, and, because detecting language impact the tokenization time, it skips the Language detection.